samples/HCL/example.tf

```diff
@@ -1,13 +1,145 @@
-resource "aws_instance" "web" {
```

Comment:
We prefer to keep sample files in their original form. They are used to train the Bayesian classifier and for tests and should reflect as best as possible actual files on github.com.
You can replace this sample entirely with a more realistic one or simply add a new one. We prefer sample files from real application (vs. sample files written as examples).